eodag/plugins/search/base.py

```diff
@@ -157,3 +159,121 @@ def get_metadata_mapping(
         return self.config.products.get(product_type, {}).get(
             "metadata_mapping", self.config.metadata_mapping
         )
+
+    def set_sort_by_arg(self, kwargs: Dict[str, Any]) -> Optional[SortByList]:
+        """Extract the "sortBy" argument from the kwargs or set it to the provider default sort configuration if needed
+
+        :param kwargs: Search arguments
+        :type kwargs: Dict[str, Any]
+        :returns: The "sortBy" argument from the kwargs or the provider default sort configuration
+        :rtype: :class:`~eodag.types.search_args.SortByList`
+        """
+        # remove "sortBy" from search args if exists because it is not part of metadata mapping,
+        # it will complete the query string or body once metadata mapping will be done
+        sort_by_arg_tmp = kwargs.pop("sortBy", None)
+        sort_by_arg = sort_by_arg_tmp or getattr(self.config, "sort", {}).get(
+            "sort_by_default", None
+        )
+        if not sort_by_arg_tmp and getattr(self.config, "sort", {}).get(
+            "sort_by_default", None
+        ):
+            logger.info(
+                f"{self.provider} is configured with default sorting by '{sort_by_arg[0][0]}' "
+                f"in {'ascending' if sort_by_arg[0][1] == 'ASC' else 'descending'} order"
+            )
+        return sort_by_arg
+
+    def transform_sort_by_arg_for_search_request(
```

Comment:
please rename to `build_sort_by` and return `(sort_by_qs: str, {}) or ("", sort_by_qp: dict)`. This will avoid `assert sort_by_provider_keyword is not None` at the end, and make it more easy to use in qssearch